content/blog/chif-driver-not-found.md

```diff
@@ -0,0 +1,115 @@
+
+The result of this in-band configuration is a `ChifDriverMissingOrNotFound` error. 
+
+> NOTE: Only the last two lines of stack dump are displayed for clarity.
+
+![blobstore target URL](/img/blobstoretarget.png "blobstore target URL")
+
+### Note concerning HPE PowerShell Cmdlets
+
+ As of the writing of this article, only the [HPESysinfoCmdlets](https://www.powershellgallery.com/packages?q=HPESysinfoCmdlets) uses the CHIF driver for in-band management. Other PowerShell Cmdlets like the [HPEiLOCmdlets](https://www.powershellgallery.com/packages/HPEiLOCmdlets/) or the [HPERedfishCmdlets](https://www.powershellgallery.com/packages/HPERedfishCmdlets/) cannot be configured for in-band management through the CHIF driver.
+
+### Miss configured HPE/VMware infrastructures
+
+I've seen cases where VMware system managers installed manually a `.rpm` iLOrest package in their ESXi hypervisor instead of getting it installed properly with a [supported HPE custom image](https://vibsdepot.hpe.com/). It is important to remember that HPE bundles and tests specific application packages of iLOrest, SUM, SUT and the Agentless Management Service (AMS) for the ESXi hypervisor. Manual installation of those applications will generate problems like the `Chif driver not found` error.
+
+## Other related problematic situations
+
+The previous sections of this article tried to explain the root causes of the `Chif driver not found` error. However you can face situations involving the CHIF driver where iLOrest commands return a valid output, but from the wrong system !
+
+### iLOrest scripts accessing a wrong iLO
+
+In this context, you are a privileged user launching an iLOrest script from a iLO based server, targeting a remote iLO. However, the script supplies wrong credentials to the remote iLO. In that case, iLOrest returns an error code, but sadly enough, the script does not test the return code of the login process and continues its execution. 
+
+As you are privileged on a physical iLO based system with the CHIF driver up and running, iLOrest performs following requests successfully on your system via the CHIF driver in place of the remote server. This situation is embarrassing because you perform management actions on the wrong system.
+
+The following picture displays a script launched from an iLO 4 based computer, targeting an iLO 5, but supplying faulty username and password in the login command and discarding errors (`&> /dev/null`).
+
+As a consequence, the next iLOrest command retrieves the local iLO 4 firmware version, instead of the remote iLO 5 firmware version ! 
+
+![iLOrest accessing wrong iLO](/img/wrongchif.png "iLOrest accessing wrong iLO")
+
+## Take away
+
+The most important things you should remember concerning the `Chif driver not found` error are:
```

Comment:
You might want to beef up this summary a little bit, maybe saying something like:

I hope you found this tutorial helpful. You can find many articles regarding iLO here in the [HPE DEV blog](https://developer.hpe.com/blog). In this post, I've tried to point out situations where you may come across the Chif driver not found error and offer suggestions on how to avoid this. The most important things you should remember concerning this error are: (and then continue as you have it there)